microsoft-edge/web-we-want/index.md

```diff
@@ -0,0 +1,42 @@
+---
+ms.assetid: 
+description: A cross-browser initiative focused on identifying missing features and functionalities in the web platform for potential development as web standards or browser features.
+title: Web We Want overview
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/15/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge, web development, html, css, developer, web we want
+---
+# The Web We Want initiative
+
+The Web We Want initiative is a cross-browser, open initiative focused on identifying missing features and functionalities in the web platform for potential development as web standards or browser features.
+
+If you build for the web, you inevitably run into problems. Perhaps there’s no way to achieve an aspect of your design with CSS, or it may require an incredible amount of hacking CSS. Maybe there’s a device feature you want to tap into using JavaScript, or there’s a needed Developer Tools feature that can make your job easier.
+
+The Web We Want initiative is the place to let browser vendors and standards groups know what you think is missing from the web platform. Here’s how you can get involved:
+
+Visit [the Web We Want][WebWeWant] website and submit web platform or developer tools features.
+
+View the [list of submitted Wants][ListOfSubmittedWants] to upvote existing Wants that you need, which helps us prioritize them.
+
+Join in the [discussions][WebWeWantDiscussions] to provide feedback, link to resources, and otherwise help usher a Want into reality.
+
+Stay in touch by following the initiative on [Twitter][TwitterWebWeWant] or [LinkedIn][LinkedInWebWeWant] to keep up-to-date on what’s in development, or to participate in surveys that help us prioritize submitted Wants.
+
+We’ve successfully shipped a few Wants into the Chromium DevTools, such as the [Source Order Viewer][SourceOrderViewer] and a 3D Viewer, similar to what was in the Firefox DevTools. There are also a handful of other Wants that have previous proposals in standards groups. Join us in driving the future of the web forward with the Web We Want!
```

Comment:
@ststimac - do we want to link to the DevTools topics for Source Order Viewer and 3D viewer?

Reply:
In case we want to link to the DevTools topics for Source Order Viewer and 3D viewer:
1. What's the URL for the Source Order Viewer?  I'm not seeing that listed in the TOC.
2. Is the following the URL for the 3d viewer?
3D View - Microsoft Edge Development | Microsoft Docs 
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/3d-view/

Reply:
Linked.